llama_stack/providers/remote/vector_io/chroma/chroma.py

```diff
@@ -16,12 +16,13 @@
 from llama_stack.apis.vector_dbs import VectorDB
 from llama_stack.apis.vector_io import Chunk, QueryChunksResponse, VectorIO
 from llama_stack.providers.datatypes import Api, VectorDBsProtocolPrivate
+from llama_stack.providers.inline.vector_io.chroma import ChromaVectorIOConfig as RemoteChromaVectorIOConfig
```

Comment:
Just one more small change - this should be aliased to `InlineChromaVectorIOConfig` instead of the Remote variant. And, down below, the alias for the `from .config import ...` one should be `RemoteChromaVectorIOConfig`. Basically, just switch the two import aliases. The logic further down looks correct, once the import aliases get switched.